docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx

```diff
@@ -0,0 +1,51 @@
+## System Clock Offset
+
+The `systemClockOffset` can optionally be passed to the `FaceLivenessDetectorCore` config, where it will be used added an offset in milliseconds to the AWS server time. This can be useful when the user's device time is inaccurate or when the user is frequently switching time zones. An example of calculating the offset and passing it to the `FaceLivenessDetectorCore` is shown below:
+
+```jsx
+import React from 'react';
+import { FaceLivenessDetectorCore } from '@aws-amplify/ui-react-liveness';
+
+function MyComponent() {
+  const [createLivenessApiData, setCreateLivenessApiData] = React.useState(null);
+  
+  /* 
+   * 1. Check whether the difference between server time and device time
+   * is greater than 5 minutes, and if so, return the offset in milliseconds
+   * This logic should be adjusted based on the server response and your use case
+  */ 
+  const getSystemClockOffset = (serverDate: string | undefined) => {
```

Comment:
This is working code but may or may not work out of the box depending on the customer's API call to createLiveness. Curious what people think about including this versus pseudocode or just function headers with comments.

Reply:
I think i'm okay adding it as an example but calling out that it depends on them having headers with date available